src/tests/auth.setup.js

```diff
@@ -1,9 +1,9 @@
-import { test as setup } from '@playwright/test';
-import { loginPage, inventoryPage} from '../fixture';
+//import { test as setup } from '@playwright/test';
```

Comment:
Lets remove this line since we dont need it

Reply:
Updated